VogtPayroll6/Employee.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll6
+{
+    class Employee
+    {
+        private int empNumber;
+        private int empHoursWorked;
+        private decimal empPayrate;
+        private decimal empTaxRate;
+
+        public int EmpNumber { get => empNumber; }
+        public int EmpHoursWorked { get => empHoursWorked; }
+        public decimal EmpPayrate { get => empPayrate; }
+        public decimal EmpTaxRate { get => empTaxRate; }
+
+        public Employee(int empNumber, int empHoursWorked, decimal empPayrate, decimal empTaxRate)
+        {
+            this.empNumber = empNumber;
+            this.empHoursWorked = empHoursWorked;
+            this.empPayrate = empPayrate;
+            this.empTaxRate = empTaxRate;
+        }
+
+        public void DisplayEmployeeInfo(Employee[] empArray)
+        {
+            Console.Write("Please enter employee number: ");
+            var empNum = Convert.ToInt32(Console.ReadLine());
+            var temp = empArray[empNum];
+            if (empArray[empNum].EmpNumber == empNum)
```

Comment:
This is a function on the employee object. Why are you passing in array? Just print the employee info thats here, available in the class.